canonical-web-and-design/canonical.com
# Add apply for a general job functionality

## Done
- Add apply for a general job functionality from the `send your CV` tab on any `careers` page

## QA
- Run the website locally using `./run`
- Go to [/careers/all](http://0.0.0.0:8002/careers/all) and click on the `Send your CV` tab
- Fill in all the fields, except the ones after CV attachment and click `Send now` button at the bottom of the page
- Check if the job application was successful on greenhouse

P.S. Ask @solazio for API_KEY in order to get this to work

## Issue

https://github.com/canonical-web-and-design/web-squad/issues/1605
